lib/be_protected/response/verification.rb

```diff
@@ -13,6 +13,10 @@ def error?
         verifications.any? && verifications.values.any?(&:error?)
       end
 
+      def has_action?(action)
+        !rules.nil? && rules.has_action?(action)
```

Comment:
`!rules.nil?` looks bad, in my opinion.
Maybe, use simple `rules && rules.has_action?(action)`?